docs/sql-reference/sql-functions/Window_function.md

```diff
@@ -691,6 +731,50 @@ from int_t
 where property in ('prime','square');
 ~~~
 
+### PERCENT_RANK()
+
+The PERCENT_RANK() function calculates the relative rank of a row within a result set as a percentage. It returns the percentage of partition values less than the value in the current row, excluding the highest value. The return values range from 0 to 1. This function is useful for percentile calculations and analyzing data distribution.
+
+The PERCENT_RANK() function is calculated using the following formula, where rank represents the row rank and rows represents the number of partition rows:
+
+~~~Plain Text
+(rank - 1) / (rows - 1)
+~~~
+
+Syntax:
+
+~~~SQL
+PERCENT_RANK() OVER (partition_by_clause order_by_clause)
+~~~
+
+The following example shows the relative rank of column y within each group of column x.
+
+~~~SQL
+SELECT x, y,
+    PERCENT_RANK()
+        OVER (
+            PARTITION BY x
+            ORDER BY y
+        ) AS `percent_rank`
+FROM int_t;
+~~~
+
+~~~Plain Text
++---+---+-----------------------+
+| x | y | percent_rank          |
++---+---+-----------------------+
+| 1 | 1 | 0.0000000000000000    |
```

Comment:
shall we reduce the precision? like `0.12` ? 
what's the behavior of other dbs ?

Reply:
MySQL maintains precision up to 16 decimal places. However, when the value is 0.0000000000000000, MySQL displays it as 0. As for this PR, the code logic is accurate, I will fix the document.

Reply:
hi @Moonm3n , could you please rebase the code?

Reply:
> 

Done, PTAL.